src/index.html

```diff
@@ -22,6 +22,393 @@
+    <header class="header">
+      <div class="header__wrap">
+        <a href="/">
+          <img
+            class="header__logo"
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
+              <a
+                class="nav__link is-active"
+                href="/"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+              >
+                Samsung
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+              >
+                Smartphones
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+              >
+                Gadgets
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+              >
+                Tablets
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+              >
+                Photo
+              </a>
+            </li>
+
+            <li>
+              <a
+                class="nav__link"
+                href="/"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="/src/images/imac.jpeg"
+          alt="card-img"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__id">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <a
+            class="card__reviews"
+            href="#"
+          >
+            Reviews: 5
+          </a>
+        </div>
+        <div class="card__price">
+          <p class="card__label">Price:</p>
+          <p class="card__value">$2,199</p>
+        </div>
+        <a
+          class="card__link"
+          href="#"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```